src/System.Private.ServiceModel/tests/Common/Scenarios/Endpoints.cs

```diff
@@ -145,6 +145,14 @@ public static string Tcp_VerifyDNS_Address
         }
     }
 
+    public static string Tcp_VerifyDNS_HostName
+    {
+        get
+        {
+            return BridgeClient.GetResourceHostName("WcfService.TestResources.TcpVerifyDNSResource");
+        }
+    }
+
```

Comment:
Agree to discuss naming.  Endpoints offer just offer services or certificates.  They don't "verify". NIT